service/org.nemomobile.BtrfsBalancer.conf

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE busconfig PUBLIC
+  "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+  "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<!-- This file goes to /etc/dbus-1/system.d -->
+<busconfig>
+  <!-- allow super user -->
+  <policy user="root">
+    <allow own="org.nemomobile.BtrfsBalancer"/>
+    <allow send_destination="org.nemomobile.BtrfsBalancer"/>
+    <allow send_interface="org.nemomobile.BtrfsBalancer"/>
+  </policy>
+
+  <!-- allow privileged users -->
+  <policy user="nemo">
```

Comment:
Yes, D-Bus can't filter on setgid. Instead, look here: https://github.com/nemomobile/password-manager/blob/master/src/passwordmanager.cpp#L149 (feel free to copy that code)